rfcs/text/0001-rfc-process.md

```diff
@@ -211,6 +283,48 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
   ***@channel**: DCP PM declined the RFC for the RFC Process - https://github.com/HumanCellAtlas/dcp-community/pull/27*
 
+**ADDITION BEGINS**
+### Pausing RFCs
+
+#### Author(s):
+
+- If via various discussions, it is decided that the RFC must be "paused" until further information can be obtained, and rough consensus has been met to pause rather than reject or withdraw, replace "rfc-community-review" with "rfc-paused" and comment in the PR with the rationale.
```

Comment:
Additional background - [Rust Postponed](https://github.com/rust-lang/rfcs#rfc-postponement) and PEP [status](https://www.python.org/dev/peps/pep-0001/) such as "Deferred". Should there be a deadline (like a Release milestone) for unpausing or closing?

Reply:
Great question. I don't have a suggested timeline at the moment so I'm inclined to (for now) take the Rust approach and "wing it." If there becomes a time that there's a backlog of paused RFCs, we can reconsider.

What do you think?